pkg/controllers/management/cis/scheduled_scan.go

```diff
@@ -255,17 +255,25 @@ func cleanOldScans(
 	clusterScanClient v3.ClusterScanInterface,
 	clusterScanLister v3.ClusterScanLister,
 ) error {
+	logrus.Debugf("scheduledScanHandler: cleanOldScans: checking")
```

Comment:
do we need this log? Does not seem to log any debug information

Reply:
will clean up.